e2e/contracts/testdappempty/TestDAppEmpty.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
```

Comment:
Consider adding a public variable to assert the contract is being called 
` bool public wasCalled;`

Reply:
We already have test for the logical side, prefer to keep this one as minimal as possible